home/ivp-planning/user-interface/the-3d-panel.md

```diff
@@ -1,28 +1,31 @@
 ---
 description: >-
-  The 3D-Panel acts primarily as visualization area. Like in the 2D-panel it
-  where you can place, move, scale and rotate machines and objects as if you
-  would work with a traditional floor plan on paper.
+  The 3D panel primarily serves as a visualization area. Similar to the 2D panel, it allows you to position, adjust size, and rotate machinery and objects as if you were working with a traditional paper floor plan.
 ---
 
 # 3D-Panel
 
 The 3D-Panel acts as a visualization area where you can work in a three dimensional representation of the room or factory. It is possible to [place](../machines/first-steps-with-3d-object.md), [move](../machines/selecting-and-moving-objects.md), [scale and rotate](../machines/scale-and-rotate-objects.md) machines and objects like in the 2D-panel. However, the 3D-panel also offers functions to take [screenshots](../advanced-tools/creating-screenshots.md) or to work with [virtual cameras](virtual-camera-panel.md) to [record videos](../advanced-tools/virtual-cameras.md).
 
```

Comment:
The first sentence is redundant with the description above. Taking screenshots and working with virtual cameras is also possible in the 2D panel. Do we really need to mention these features here? This just makes the documentation get outdated faster. I think this paragraph and the description above could be heavily trimmed down.